src/core/webidl-clipboard.js

```diff
@@ -6,16 +6,30 @@
  * well-formatted IDL to the clipboard.
  *
  */
-import svgClipboard from "text!../../assets/clipboard.svg";
+import { fetchAsset } from "./text-loader.js";
 export const name = "core/webidl-clipboard";
 
-// This button serves a prototype that we clone as needed.
-const copyButton = document.createElement("button");
-copyButton.innerHTML = svgClipboard;
-copyButton.title = "Copy IDL to clipboard";
-copyButton.classList.add("respec-button-copy-paste", "removeOnSave");
+const copyButtonPromise = createButton();
+
+async function loadSVG() {
```

Comment:
I wonder if we should generalize this into a util function... `loadTextAsset()` or something. I imagine we eventually want to kill these methods once we address whatever is requiring them.

Reply:
That's essentially https://github.com/w3c/respec/pull/2553#discussion_r339940372, and we can't remove it as I replied there:

>No, dynamic import with dynamic value will prevent bundler from merging assets into one file, because they are not smart enough 😂